article/app/services/dotcomponents/ArticlePicker.scala

```diff
@@ -123,13 +123,17 @@ object ArticlePageChecks {
       "tone/albumreview",
       "tone/analysis",
       "tone/blog",
+      "tone/callout",
+      "tone/childrens-user-reviews",
       "tone/comment",
       "tone/competitions",
       "tone/documentaries",
       "tone/editorials",
+      "tone/event-descriptions",
       "tone/explainers",
       "tone/extract",
       "tone/features",
+      "tone/graphics",
```

Comment:
Not used often but usually contain interactive embeds which are not supported anyway.